examples/cookbook/package.json

```diff
@@ -12,6 +12,7 @@
     "chalk": "^4.1.1",
     "homedir": "^0.6.0",
     "near-api-js": "^0.41.0",
-    "path": "^0.12.7"
+    "path": "^0.12.7",
+    "tweetnacl": "^1.0.3"
```

Comment:
these can be removed.

- `tweetnacl` is no longer used
- `path` is built into node